src/optimizer/query_to_operator_transformer.cpp

```diff
@@ -26,6 +26,7 @@
 #include "parser/statements.h"
 
 #include "catalog/manager.h"
+#include "common/exception.h"
```

Comment:
Is this needed?

Reply:
We can still catch `CatalogException` without it, so I think it isn't needed. I included it while adding the test just to be safe. I'll remove it.